src/components/frame/LandingPage/InfoBanner.tsx

```diff
@@ -0,0 +1,97 @@
+import { Password, Confetti, Keyhole } from '@phosphor-icons/react';
+import clsx from 'clsx';
+import React from 'react';
+
+import LoadingSkeleton from '~common/LoadingSkeleton/LoadingSkeleton.tsx';
+import { formatText } from '~utils/intl.ts';
+
+export interface InfoBannerProps {
+  variant?: 'info' | 'success' | 'error';
+  loading?: boolean;
+  title: string;
+  text: string;
+}
+
+const displayName = 'frame.LandingPage';
+
+const InfoBanner = ({
+  variant = 'info',
+  loading,
+  title,
+  text,
+}: InfoBannerProps) => {
+  const renderIcon = () => {
+    switch (variant) {
+      case 'success':
+        return <Confetti size={20} className="text-success-400" />;
```

Comment:
What if we just pass in the `Icon` in like we do it [here](https://github.com/JoinColony/colonyCDapp/blob/master/src/components/v5/shared/NotificationBanner/NotificationBanner.tsx) :eyes: ?